snuba/datasets/configuration/generic_metrics/storages/counters_meta.yaml

```
@@ -0,0 +1,59 @@
+version: v1
+kind: readable_storage
+name: generic_metrics_counters_meta
+storage:
+  key: generic_metrics_counters_meta
+  set_key: generic_metrics_counters
+readiness_state: complete
+schema:
+  columns:
+    [
+      { name: org_id, type: UInt, args: { size: 64 } },
+      { name: project_id, type: UInt, args: { size: 64 } },
+      { name: use_case_id, type: String },
+      { name: metric_id, type: UInt, args: { size: 64 } },
+      { name: tag_key, type: UInt, args: { size: 64 } },
+      { name: timestamp, type: DateTime },
+      { name: retention_days, type: UInt, args: { size: 16 } },
+      {
+        name: count,
+        type: AggregateFunction,
+        args:
+          {
+            func: sum,
+            arg_types: [{ type: Float, args: { size: 64 } }],
+          },
+      },
+    ]
+  local_table_name: generic_metric_counters_meta_local
+  dist_table_name: generic_metric_counters_meta_dist
+
+allocation_policies:
+  - name: ConcurrentRateLimitAllocationPolicy
+    args:
+      required_tenant_types:
+        - referrer
+        - organization_id
+        - project_id
+      default_config_overrides:
+        is_enforced: 0
+  - name: BytesScannedWindowAllocationPolicy
+    args:
+      required_tenant_types:
+        - referrer
+        - organization_id
+      default_config_overrides:
+        is_enforced: 1
+        throttled_thread_number: 1
+        org_limit_bytes_scanned: 100000
```

This number is probably too low

```suggestion
        org_limit_bytes_scanned: 10000000
```